backend/src/main/java/com/javahelp/backend/query/IConstraint.java

```diff
@@ -0,0 +1,11 @@
+package com.javahelp.backend.query;
+
+import java.util.ArrayList;
+import java.util.Map;
+
+public interface IConstraint {
```

Comment:
Does this interface add value beyond the abstract class? Will there ever be a `IConstraint` that does not extend the abstract class?

Reply:
Agreed. Could be deleted.